src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -750,122 +766,126 @@ void func_80B868B8(EnKaizoku* this, PlayState* play) {
+            this->subCamEyeTarget.z = player->actor.world.pos.z - 71.0f;
 
-            this->unk_5D4.x = player->actor.world.pos.x - 110.0f;
-            this->unk_5D4.y = player->actor.world.pos.y;
-            this->unk_5D4.z = player->actor.world.pos.z + 30.0f;
-            this->unk_5A0++;
+            this->subCamAtTarget.x = player->actor.world.pos.x - 110.0f;
+            this->subCamAtTarget.y = player->actor.world.pos.y;
+            this->subCamAtTarget.z = player->actor.world.pos.z + 30.0f;
+            this->flashTimer++;
             if (curFrame >= 1.0f) {
-                Math_ApproachZeroF(&this->unk_2F8.x, 0.3f, 0.3f);
-                Math_ApproachZeroF(&this->unk_304.x, 0.3f, 0.3f);
-                this->unk_2F8.z = this->unk_2F8.y = this->unk_2F8.x;
-                this->unk_304.z = this->unk_304.y = this->unk_304.x;
+                Math_ApproachZeroF(&this->swordScaleRight.x, 0.3f, 0.3f);
+                Math_ApproachZeroF(&this->swordScaleLeft.x, 0.3f, 0.3f);
+                this->swordScaleRight.z = this->swordScaleRight.y = this->swordScaleRight.x;
+                this->swordScaleLeft.z = this->swordScaleLeft.y = this->swordScaleLeft.x;
             }
-            if (this->unk_5A0 == 0x12) {
-                Actor* temp_v0_2;
+            if (this->flashTimer == 0x12) {
+                Actor* dekuNut;
 
                 Actor_PlaySfx(&this->picto.actor, NA_SE_EN_PIRATE_SHOUT);
-                temp_v0_2 = Actor_Spawn(&play->actorCtx, play, ACTOR_EN_ARROW, this->picto.actor.world.pos.x,
-                                        this->picto.actor.world.pos.y + 10.0f, this->picto.actor.world.pos.z,
-                                        this->picto.actor.shape.rot.x, this->picto.actor.shape.rot.y,
-                                        this->picto.actor.shape.rot.z, -8);
-                if (temp_v0_2 != NULL) {
-                    temp_v0_2->gravity = -10.0f;
+                // oddly, instead of just playing the sfx and spawning an effect
+                //  they spawn a real nut
+                dekuNut = Actor_Spawn(&play->actorCtx, play, ACTOR_EN_ARROW, this->picto.actor.world.pos.x,
+                                      this->picto.actor.world.pos.y + 10.0f, this->picto.actor.world.pos.z,
+                                      this->picto.actor.shape.rot.x, this->picto.actor.shape.rot.y,
+                                      this->picto.actor.shape.rot.z, -ARROW_TYPE_DEKU_NUT);
+
+                if (dekuNut != NULL) {
+                    dekuNut->gravity = -10.0f;
                     play->envCtx.fillScreen = true;
                 }
             }
 
-            if (this->unk_5A0 >= 0x12) {
-                Math_ApproachF(&this->unk_5EC, 60.0f, 1.0f, 20.0f);
-                this->unk_5E8 = this->unk_5EC / 60.0f;
-                play->envCtx.screenFillColor[3] = this->unk_5E8 * 255.0f;
+            if (this->flashTimer >= 0x12) {
```

Comment:
iirc we write timers in dec instead of hex
```suggestion
            if (this->flashTimer >= 18) {
```
Could you do the same for the other uses too?